rdf-delta-cmds/src/main/java/delta/server/DeltaServerCmd.java

```diff
@@ -51,36 +51,32 @@
 
     static { DeltaLogging.setLogging(true); }
 
-    private static Logger LOG = Delta.DELTA_LOG;
+    private static final Logger LOG = Delta.DELTA_LOG;
 
-    private static ArgDecl argHelp              = new ArgDecl(false, "help", "h");
-    private static ArgDecl argVerbose           = new ArgDecl(false, "verbose", "v");
-    //private static ArgDecl argVersion    = new ArgDecl(false, "version");
-    private static ArgDecl argPort              = new ArgDecl(true, "port");
+    private static final ArgDecl argHelp              = new ArgDecl(false, "help", "h");
+    private static final ArgDecl argVerbose           = new ArgDecl(false, "verbose", "v");
+    private static final ArgDecl argPort              = new ArgDecl(true, "port");
 
-    private static ArgDecl argBase              = new ArgDecl(true,  "base");
+    private static final ArgDecl argBase              = new ArgDecl(true,  "base");
     // Specific choices
-    private static ArgDecl argFile              = new ArgDecl(false, "file");
-    private static ArgDecl argRocks             = new ArgDecl(false, "rocks", "rocksdb", "rocksDB");
-
-    private static ArgDecl argMem               = new ArgDecl(false, "mem");
-    private static ArgDecl argStore             = new ArgDecl(true, "store", "rdb");
-    private static ArgDecl argZk                = new ArgDecl(true, "zk");
-    private static ArgDecl argZkPort            = new ArgDecl(true, "zkPort", "zkport");
-    private static ArgDecl argZkData            = new ArgDecl(true, "zkData", "zkdata");
-    private static ArgDecl argZkConf            = new ArgDecl(true, "zkCfg", "zkcfg", "zkConf", "zkconf");
-
-    private static ArgDecl argS3Bucket          = new ArgDecl(true, "s3bucket");
-    private static ArgDecl argS3Region          = new ArgDecl(true, "s3region");
-    private static ArgDecl argS3KeysFile        = new ArgDecl(true, "s3keys");
-    private static ArgDecl argS3KeysProfile     = new ArgDecl(true, "s3profile");
+    private static final ArgDecl argFile              = new ArgDecl(false, "file");
+    private static final ArgDecl argRocks             = new ArgDecl(false, "rocks", "rocksdb", "rocksDB");
+
+    private static final ArgDecl argMem               = new ArgDecl(false, "mem");
+    private static final ArgDecl argStore             = new ArgDecl(true, "store", "rdb");
+    private static final ArgDecl argZk                = new ArgDecl(true, "zk");
+    private static final ArgDecl argZkPort            = new ArgDecl(true, "zkPort", "zkport");
+    private static final ArgDecl argZkData            = new ArgDecl(true, "zkData", "zkdata");
+    private static final ArgDecl argZkConf            = new ArgDecl(true, "zkCfg", "zkcfg", "zkConf", "zkconf");
```

Comment:
I didn't write or modify this line, so it's only being flagged because I touched a line above it. I don't want to do more than remove code that has been commented out and legitimately unused stuff.